src/content/developers/docs/bridges/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Bridges
+description: An overview of bridging for developers
+lang: en
+sidebar: true
+---
+
+With the proliferation of L1 blockchains and L2 [scaling](/developers/docs/scaling/) solutions, alongside an ever-growing number of decentralized applications going cross-chain, the need for communication and asset movement across chains has never been more important.
+
+This is where blockchain bridges come in.
+
+## Need for bridges {#need-for-bridges}
+
+Bridges exist to connect blockchain networks. They enable connectivity and interoperability between blockchains.
+
+Blockchains exist in siloed environments, meaning there is no way for blockchains to trade and communicate with other blockchains naturally. As a result, while there could be significant activity and innovation within an ecosystem, it is limited by the lack of connectivity and interoperability with other ecosystems.
+
+Bridges offer a way for isolated blockchain environments to connect with each other. They establish a transportation route between blockchains where tokens, messages, arbitrary data, and even [smart contract](/developers/docs/smart-contracts/) calls can be transferred from one chain to another.
+
+## Benefits of bridges {#benefits-of-bridges}
+
+Put simply, bridges unlock developer potential by allowing blockchain networks to exchange data and move assets between them.
+
+Different blockchains and communities have unique strengths, weaknesses, and approaches to building applications (such as speed, throughput, costliness, etc.). Bridges help the development of the overall crypto ecosystem by enabling blockchains to leverage the innovations of each other.
```

Comment:
Edit for clarity: communities don't have speed/throughput/costliness properties.

```suggestion
Blockchains have unique strengths, weaknesses, and approaches to building applications (such as speed, throughput, costliness, etc.). Bridges help the development of the overall crypto ecosystem by enabling blockchains to leverage the innovations of each other.
```

Reply:
The sentence order should be reworked. The main benefit suggested under this heading is buried.